seed/seed.json

```diff
@@ -657,6 +657,34 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"],
                 "country": ["GB"]
             }
+        },
+        {
+            "name": "MacCoreLocationBackendStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 100,
+                    "feature_association": {
+                        "enable_feature": ["MacCoreLocationBackend"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 0,
+                    "feature_association": {
+                        "disable_feature": ["MacCoreLocationBackend"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "91.1.26.74",
+                "channel": ["RELEASE", "BETA", "NIGHTLY"],
```

Comment:
will this actually work without any issues? I mean, enabling it on 100% Release is quite bold.

Reply:
I set to 100% because it looks like chrome's stable version already uses this feature.
To safe, I think testing on nightly and beta first seems fine.

Reply:
Removed `RELEASE` from channel list.